addons/html_editor/static/src/core/delete_plugin.js

```diff
@@ -331,6 +331,15 @@ export class DeletePlugin extends Plugin {
     removeNode(node) {
         const root = node;
         const remove = (node) => {
+            if (node.dataset.oeManaged) {
```

Comment:
I would call it `oeHasRemovableHandler` but I'm sure dmo will have a strong opinion about it